api/edge.py

```diff
@@ -46,7 +46,16 @@ def get_timestamp():
 
 
 def read_all():
-    return [EDGES[key] for key in sorted(EDGES.keys())]
+    db = Database()
+
+    statement = "SELECT * FROM tbl_edge"
+    print(statement)
+    edge = db.execute(statement)
```

Comment:
pirk, men kall det edges, ikke edge, metoden heter tross alt "read_all()"

Reply:
Vi burde også endre til å bruke logging i stedet for print's, men dette gjelder hele prosjektet.

Reply:
Endret edge til edges. Ser på logging etter at vi har fått på plass siste funksjonliteten.